### DISCUSSION TITLE
Why is my HTML source different with "View source" and "Inspector"

### DISCUSSION THREAD

**Discussion by PrimozRome** [vercel/next.js#40747] [Ideas]
When I check the source code of my Next.js website the source is substantially different if I use (Google Chrome):
- Left-click "View source" and
- Inspect

With "View source" I don't see all the content, internal links, etc... But when I use "Inspector" links and other HTML tags are there. I am using SSG.

What am I missing here?

**Comment by HATGAMING09** [comment]
Hi
